tst/benchmarking/benchmark_commons/test_hpo_main_local.py

```diff
@@ -25,6 +27,7 @@
 from benchmarking.nursery.benchmark_multiobjective.hpo_main import main
 
 
+@pytest.mark.skip("NEEDS FIXING")
```

Comment:
The test started failing after https://github.com/awslabs/syne-tune/pull/672 was merged.

It's a new test that I just added, I am checking now to see if we can keep it enabled or if we really need to skip it.